e2e/workspace-create/src/create-nx-workspace.test.ts

```diff
@@ -77,16 +77,6 @@ describe('create-nx-workspace', () => {
     expectNoAngularDevkit();
   });
 
-  it('should be able to create an empty workspace with ts/js capabilities', () => {
```

Comment:
Why was this test removed? You probably intended to move it to the new test file?

Reply:
i think i removed this one because it is a duplicate of the tests in e2e/workspace-create/src/create-nx-workspace-npm.test.ts, but i revert it back